src/vm/tests/assets.rs

```diff
@@ -248,6 +248,59 @@ fn test_simple_token_system() {
     
 }
 
+#[test]
+fn total_supply() {
+    let contract = "(define-fungible-token stackaroos 5)
```

Comment:
Should there also be a test for `(define-fungible-token stackaroos -5)`?

Reply:
Yeah, will add. Will add a type enforcement test for the parameter too.

Reply:
Added in ac07bc2